tests/reals/real_doublylinkedlist.c

```diff
@@ -3,6 +3,5 @@
 
 #include "real_doublylinkedlist_renames.h"
 
-#define GBALLOC_H
```

Comment:
>GBALLOC_H [](start = 8, length = 9)

doesn't need gballoc renames?

Reply:
doesn't. doublylinkedlist is malloc free completely.

---
In reply to: [463691423](https://github.com/Azure/azure-c-util/pull/55#discussion_r463691423) [](ancestors = 463691423)